HW1/P3/P3.py

```diff
@@ -0,0 +1,10 @@
+from pyspark import SparkContext
+
+sc = SparkContext(appName="P3")
+words = sc.textFile('wlist.txt')
+counted_words = words.map(lambda x: (''.join(sorted(x)), [x])) \
+								.reduceByKey(lambda words1, words2: words1 + words2) \
+								.map(lambda (sorted_word, anagrams): (sorted_word, len(anagrams), anagrams)) \
+								.sortBy(lambda (sorted_word, count, anagrams): count, ascending=False)
+
+print counted_words.take(1)[0]
```

Comment:
It looks very complicated to write all operations in one line. Maybe split to several lines and add some comments is better.